### PR TITLE
Feature/layout mobile

### PR DESCRIPTION
**Pull request by WitchDevelops**
I suggest we merge now, as there are some merge conflicts. Better to resolve them sooner than later. After merging we can continue working on the layout :)

I've added dimensions to the main card and styled fonts based on the Figma file.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -50,8 +50,9 @@ img, picture {
   --n-blue-dark: hsl(218, 23%, 16%); /* neutral: dark blue */
 
   /* fonts */
-  --fs-quote: 1.75rem; /* font size: 28px */
-  --fs-title: 1.75rem;
+  --fs-quote-d: 1.75rem; /* font size: 28px on dekstop */
+  --fd-quote-m: 1.5rem; /* font size: 24px on mobile */
```
Comment (caesiumtea) [inline style.css:54]:
```suggestion
  --fs-quote-m: 1.5rem; /* font size: 24px on mobile */
```
looks like a typo haha

Comment (WitchDevelops) [reply style.css:54]: eagle eye!

Comment (caesiumtea) [reply style.css]: I was just stumped trying to figure out what fd stood for...

**Review by caesiumtea — APPROVED** [review]

**Comment by caesiumtea** [comment]
I'm approving this because of what we said about just getting the site _working_ first and then worrying about refactoring and improving afterward, but I feel like this isn't quite the right way to size the card. Check out what happens on intermediate screen sizes:
image
I don't think that having such a wide card is consistent with the intention of the Frontend Mentor design. I wonder if we want the card to "shrink wrap" around the divider image (with some padding in between), instead of being a defined width?

I like the ideas that Carlos suggested here for how to lay out the card: https://github.com/commIT-Group/Advice-generator-app/discussions/31

You can also see what you think about this offshoot branch where I tested a different layout method, where I measured just the size of the padding in the example image, and did the spacing with grid properties and margins and paddings. https://github.com/commIT-Group/Advice-generator-app/commit/903b41ae93eab5485a07cd3babcbc7f8ad08613f

**Comment by caesiumtea** [comment]
Closes issue #26

**Review by nikkehtine — APPROVED** [review]

**Pull request merged**